e2e-playwright/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "e2e-playwright",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "install:browsers": "playwright install --with-deps",
+    "test": "playwright test",
+    "test:chrome": "playwright test --project=chromium",
+    "test:firefox": "playwright test --project=firefox",
+    "test:safari": "playwright test --project=webkit",
+    "test:debug": "playwright test --debug",
+    "lint": "eslint \"./**/*.ts\" && echo \"lint success\"",
+    "lint:fix": "eslint --fix \"./**/*.ts\" && echo \"lint success\"",
+    "clean": "rm -rf test-results && rm -rf test-report",
+    "serve": "http-server --proxy $(grep PLAYWRIGHT_HOST .env | cut -d '=' -f2)? ../build --port 1337"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "dotenv": "16.0.3",
+    "typescript": "4.9.4"
+  },
+  "devDependencies": {
+    "@playwright/test": "1.28.1",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-playwright": "0.11.2",
```

Comment:
I am wondering if we could use `overrides` in root .eslintrc instead of creating it again in this sub-directory.
Anyway this is minor and out-of-scope of this PR but maybe worth to thing about it in a future.

Reply:
This is what i tried at first. I gave up on it because root eslint has rules for JEST, which conflicted with playwright syntax. Anyway, i tried it on old root eslint (because i at first i used old version of our starter, pulled from bitbucket :|), on newest starter it works (because of provided path `files` in `overrides`) without eslint conflits :) 

So, what i changed: still left separated .eslintrc for e2e-playwright, because i think eslint-plugin-playwright should be installed only for e2e-playwright package.json, not for root package.json.
And in e2e-playwright eslint i just extends the root eslint config and extends eslint-plugin-playwright config. WDYT about that?

Reply:
sounds good!